devfile.yaml

```diff
@@ -8,7 +8,7 @@ projects:
       location: 'https://github.com/eclipse/che.git'
 components:
   - type: cheEditor
-    id: eclipse/che-theia/1.0.0
+    id: eclipse/che-theia/next
```

Comment:
Che Server automatically provisions Theia along with exec plugin. We can make this devfile a bit simpler and remove these components here.

Reply:
yes